x-pack/plugins/maps/server/maps_telemetry/collectors/register.ts

```diff
@@ -19,7 +17,7 @@ export function registerMapsUsageCollector(
   }
 
   const mapsUsageCollector = usageCollection.makeUsageCollector({
-    type: TELEMETRY_TYPE,
+    type: 'maps',
```

Comment:
@Bamieh Unfortunately you'll need to change this back to `maps-telemetry` before merging due to this issue: https://github.com/elastic/kibana/issues/69869 :frowning: